conanfile.py

```diff
@@ -200,17 +210,24 @@ def get_configure_command_args(self):
         if not self.options.with_ldap:
             params.append("--disable-ldap")
 
-        if self.options.custom_cacert:
-            params.append('--with-ca-bundle=cacert.pem')
+        if self.options.with_ca_bundle == False:
+            params.append("--without-ca-bundle")
+        elif self.options.with_ca_bundle:
+            params.append(f"--with-ca-bundle={str(self.options.with_ca_bundle)}")
```

Comment:
Conan requires Python 3.5 that does not have f-string syntax which was introduced in 3.6.

```suggestion
            params.append("--with-ca-bundle=" + self.options.with_ca_bundle)
```

Reply:
Bummer! I've kept the `str()` sine the option type is any.